impl/sql/mutationstorage/queue_test.go

```diff
@@ -37,24 +37,22 @@ func TestRandShard(t *testing.T) {
 
 	for _, tc := range []struct {
 		desc       string
-		send       bool
+		send       []int64
 		wantCode   codes.Code
 		wantShards map[int64]bool
 	}{
 		{desc: "no rows", wantCode: codes.NotFound, wantShards: map[int64]bool{}},
-		{desc: "second", send: true, wantShards: map[int64]bool{
+		{desc: "second", send: []int64{1, 2, 3}, wantShards: map[int64]bool{
```

Comment:
Can you add another case for single shard?
Are there restrictions on shard IDs themselves (e.g. they should be positive)?

Reply:
Added test.
shardIDs can be any unique number.